sidecar-injector/README.md

```diff
@@ -13,101 +31,199 @@ The result should be:
+
+### You can check and inspect the CSR
+kubectl describe csr "vigilant-numbat-cyberark-sidecar-injector.injectors"
+
+### Approve the CSR
+kubectl certificate approve "vigilant-numbat-cyberark-sidecar-injector.injectors"
+
+Now that everything is setup you can enjoy the Cyberark Sidecar Injector.
+This is the general workflow:
+
+1. Annotate your application to enable the injector and to configure the sidecar (see README.md)
+2. Webhook intercepts and injects containers as needed
+
+Enjoy.
+
+```
+
+Make sure to read the NOTES section once the chart is installed; instructions are provided on how to accept the CSR request. The CSR request **must** be approved.
+
+## Using the Sidecar Injector
+
+### Configuration
+
+The sidecar injector will not inject the sidecar into pods by default. Add the `sidecar-injector.cyberark.com/inject` annotation with value `true` to the pod template spec to enable injection.
+
+The following table lists the configurable parameters of the Sidecar Injector and their default values.
+
+| Parameter                     | Description                                     | Default                                                    |
+| -----------------------       | ---------------------------------------------   | ---------------------------------------------------------- |
+| `sidecar-injector.cyberark.com/inject`| Enable the Sidecar Injector by setting to `true`            | `nil` (required) |
+| `sidecar-injector.cyberark.com/secretlessConfig` | ConfigMap holding Secretless configuration               |  `nil` (required for secretless)  |  
+| `sidecar-injector.cyberark.com/conjurAuthConfig` | ConfigMap holding Conjur authentication configuration            |  `nil` (required for authenticator |
+| `sidecar-injector.cyberark.com/conjurConnConfig` | ConfigMap holding Conjur connection configuration               |  `nil` (required for authenticator |
+| `sidecar-injector.cyberark.com/injectType` | Injected Sidecar type (`secretless` or `authenticator`)                    |  `nil` (required) |
+| `sidecar-injector.cyberark.com/containerMode` | Sidecar Container mode (`init` or `sidecar`)                  |  `nil` (only applies to authenticator) |
+| `sidecar-injector.cyberark.com/containerName` | Sidecar Container name                  |  `nil` (only applies to secretless)                              |  
+
+#### sidecar-injector.cyberark.com/secretlessConfig
+
+Expected to contain the following path:
+
++ secretless.yml - Secretless Configuration File
 
-For this section, you'll work from a test namespace (test-namespace). Later you will label this namespace with `secretless-sidecar-injector=enabled` so as to allow the secretless-sidecar-injector to operate on pods created in this namespace.
+#### sidecar-injector.cyberark.com/conjurConnConfig
+
+Expected to contain the following paths:
+
++ CONJUR_VERSION - the version of your Conjur instance (4 or 5)
```

Comment:
We should put down what this defaults to

Reply:
CONJUR_VERSION is a requirement and does not default to anything right now. better to do that in another PR